homeassistant/components/light/intent.py

```diff
@@ -27,6 +27,9 @@ async def async_setup_intents(hass: HomeAssistant) -> None:
             INTENT_SET,
             DOMAIN,
             SERVICE_TURN_ON,
+            required_slots={
+                vol.Required(vol.Any("color", "temperature", "brightness")): str,
+            },
```

Comment:
Note: I suspect that `color` and `temperature` are mutually exclusive, but i'm not sure. If they are, I could model that too in another PR.